pkgs/by-name/mi/mission-center/package.nix

```diff
@@ -133,6 +145,12 @@ stdenv.mkDerivation rec {
     patchShebangs data/hwdb/generate_hwdb.py
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${builtins.placeholder "out"}/bin/${meta.mainProgram}";
```

Comment:
placeholder is like map available without any prefix.

Reply:
?

Reply:
> placeholder is like map available without any prefix.

Was that a suggestion ?

Reply:
I'm also confused with Sandro's comment.

Reply:
You can just use placeholder like map without the builtins. prefix.

Reply:
Fair enough, now I understand. Thank you Sandro.